### DISCUSSION TITLE
簡單介紹AI的一系列影片

### DISCUSSION THREAD

**Discussion by brian431** [1101-GS3073-C/Python-Community-GS3073-C#467] [General]
跟我昨天發的那個同一頻道也有做一系列AI的介紹影片。若有興趣的同學可以參考參考，我還沒看。

https://youtu.be/GvYYFloV0aA

**Comment by manipulateed** [comment]
雖然是英文，但卻能深入淺出的講解，是個很好的教材，感謝分享。

**Comment by boson13579** [comment]
看完了preview，可以發現這是一系列對ai的介紹，但是全英文對我還是太困難了，選擇放棄

**Comment by Chenszuhong** [comment]
這感覺是一個很有趣的議題，值得學習
